Documentation/ReleaseNotes.md

```diff
@@ -0,0 +1,283 @@
+package (Demos\HandInteraction\Scenes\HandInteractionDemos.unity) for a demonstration of using the
+SystemKeyboardExample script.
+
+**Take Screenshot**
+
+A Take Screenshot utility menu item (Mixed Reality Toolkit > Utilities > Take Screenshot) has been added
+to capture high resolution screenshots within the editor. Screenshots can be captured with a transparent
+clear color for use in easy post compositing of images for documentation or media.
+
+**UX Controls**
+
+The following UX controls have been added and/or improved in version 2.0.0. In addition, those in the
+following list can now be instantiated and configured from code.
+
+- BoundingBox
+- ManipulationHandler
+- HandInteractionPanZoom
+- Interactable (basic features)
+
+**Updated Architecture Documentation**
+
+The [archtecture documentation](Architecture/Overview.md) is all new for version 2.0.0.
+
+### Known issues in 2.0.0
+
+The sections below highlight some of the known issues in the Microsoft Mixed Reality Toolkit
+2.0.0.
+
+**Extension Service Wizard**
+
+When using the Extension Service Wizard,  *Generate Inspector* and/or *Generate Profile* are not actually optional. Trying to create an extension service with either of these deselected will result in an error on the following page. Furthermore, the extension service created for the user will create a property for the ScriptableObject profile that was not actually created. This results in a compiler error until the property line is removed. 
+
+Current workaround steps:
+
+1. Ignore error message in Extension Service Wizard
+1. Open up the *ExtensionService.cs file created and remove reference to the non-existent profile.
+
+Issue [#5654](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/5654) is tracking this problem.
+
+**Runtime profile swapping**
+
+MRTK 2.0.0 does not fully support profile swapping at runtime. This feature is being
+investigated for a future release.
+
+**Text overflow in the some of the scroll view example when deployed with 16-bit depth buffer**
+
+Examples made with Unity's scroll view show text overflow on the device when it is built with 16-bit depth buffer. (e.g. EyeTrackingDemo-03-Navigation.unity scene) Use 24-bit depth to build these example scenes.
+
+**Unity 2019: Could not copy the file HolographicAppRemoting.dll**
+
+There is a known issue with version 3.0.0 of the Windows Mixed Reality package for Unity 2019. If
+your project has this version installed, you will see the following error when compiling in Microsoft
+Visual Studio.
+
+To work around the issues, please check for a newer version or roll back to version 3.0.2 using Window > Package Manager in the Unity editor.
+
+**VR/Immersive devices: Content in some demo scenes is placed below the user**
+
+Some demo scenes contained in the Examples package are optimized for HoloLens devices(headset's y position is 0). These scenes
```

Comment:
>devices [](start = 78, length = 7)

Nit: space after devices (same for the next line)

Reply:
fixed